docs/_static/network/v1.yaml

```diff
@@ -215,6 +215,9 @@ components:
         retries:
           description: Number of times the event has been retried.
           type: integer
+        latest:
```

Comment:
`latest` doesn't say the latest of what, ("latest event"?)

Maybe `last_publish`, `last_attempt`, `last_delivery_attempt`, `last_retry`? (I find "retry" not the best name, since the first attempt isn't a retry, I think?)

(not 100% sure about last/latest)

Reply:
Will change to `latest_notification_attempt` as it is probably the best fit considering the namespace this lives in, but it is very length . Anything containing `last` could be misinterpreted as the last time it will be attempted